spec/checkPullRequestLabelsSpec.js

```diff
@@ -67,7 +68,16 @@ describe('Pull Request Label Check', () => {
               throw new Error('User is not a collaborator.');
             }
             return { status: 204 };
-          })
+          }),
+        getCommit: jasmine.createSpy('getCommit').and.resolveTo({
+          data: {
+            commit: {
+              author: {
+                date: '2021-04-12T18:33:45Z'
+              }
+            }
+          }
+        })
```

Comment:
This should be removed from here and only added to places where it is required.

Reply:
done